css/index.css

```diff
@@ -52,7 +76,7 @@ body
       #Project{
          color: darkgreen;
          background-color: yellow;
-      }
+      } */
```

Comment:
As mentioned below, in general  you'll want to remove any commented out code before open a PR, unless there's a specific reason for it to be there.